packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -314,6 +310,19 @@ augmentFieldsDict(Tags, {
   },
 });
 
+augmentFieldsDict(TagRels, {
+  autoApplied: {
+    resolveAs: {
+      type: "Boolean",
+      resolver: async (document: DbTagRel, args: void, context: ResolverContext) => {
+        const tagBotUserId = await getTagBotUserId(context);
+        if (!tagBotUserId) return false;
+        return (document.userId===tagBotUserId && document.voteCount===1);
```

Comment:
Didn't get all the way through the logic; should tags only count as autoApplied if nobody else has voted on the tag rel yet (even after it was autoApplied by tagBot)?

Reply:
Correct; if someone else also comes along and applies the same tag, it no longer counts.